src/content/community/contributor-roles/_index.en.md

```diff
@@ -179,6 +185,8 @@ The following apply to people who would be an owner:
   * Identifying subtle or complex issues in designs and implementation PRs
 * Directly contributed to the project through implementation and / or review
 
+Project Owners can be removed by stepping down or by two thirds vote of Project Owners.
```

Comment:
This seems a bit too open to hostile takeover possibilities to me.

Reply:
@Oats87 With our current PO situation, I don't think there can be a system in which a PO can be removed and hostile takeovers are prevented. If it were "by vote of all POs other than the one being removed", it'd still be the same situation.

We'd need to do something more like a TSC I think? Open to suggestions. I'll be thinking about it.

Reply:
We could just say "POs can't be removed except for Code of Conduct violations" for now, and build a TSC when we go for CNCF Incubation.

Reply:
ODL is currently using only a Committer at Large (CaL) Represented Group (RG) with a company cap of 49%. That is quite simple, good for smaller communities, but also provides protection from takeovers. 

https://wiki.opendaylight.org/display/ODL/TSC+Election+Process#TSCElectionProcess-TSCSeatAllocationsByRepresentedGroupforthe2021elections

Reply:
I know it's complex, but tl;dr that using that system, if at least 3 people from different companies ran, each company would be guaranteed at least one seat and we would be protected from takeovers.

Reply:
The company cap makes sense. Ditching anybody of the project or doing takeovers is far from our intentions, would be a terrible start for a project ;-]

Reply:
> The company cap makes sense. 

To be clear, the company cap can't really exist without the rest of the system. Most of the complexity of the system comes from implementing the company cap while satisfying a number of other [important principles](https://wiki.opendaylight.org/display/ODL/TSC+Election+Process#TSCElectionProcess-Principles).

> would be a terrible start for a project

Right, and even if one imagines we have no individual ethics and are corporate bots it would be absurd for an open source company to willfully take the reputational damage from crushing a budding open source project. :sweat_smile:

Reply:
> ODL is currently using only a Committer at Large (CaL) Represented Group (RG) with a company cap of 49%. That is quite simple, good for smaller communities, but also provides protection from takeovers.

My intention was to propose something like this as a simple TSC-based solution, *once we make the move to a TSC*. I think there are no Sandbox projects with TSCs now, and I doubt there are many Incubation projects even.

For now I think we can just freeze changes except for CoC violations, and add some wording that we must change to adopt a TSC-based system by a year-or-so out and if not we default to ODL's system with only CaL RG and 49% company cap as described above.

Reply:
Does a TSC make sense in a single-project context? Who would we expect to be in the TSC, and how would we ensure that it wasn’t just the project owners?

Reply:
what's a TSC? :)

Reply:
> Does a TSC make sense in a single-project context?

Maybe it's better just to scope/call it "electing/removing project owners".

> Who would we expect to be in the TSC, and how would we ensure that it wasn’t just the project owners?

I imagined that by the time we go for Incubation, we would have a fairly substantial user community from which we could elect people. I'd hope that the current POs would run and be elected, in addition to new POs from other ~companies.

> what's a TSC? :)

Technical Steering Committee, elected project leadership.

Reply:
I think a current solution of freezing removals except for CoC violations makes sense, although I would say a TSC is going to require additional manpower to run; if you think that there is enough bandwidth in order to run such a committee then that would be OK, but otherwise, the ODL system with a CaL RG + 49% company cap is "simplest" past the CoC violation removal.

Reply:
@Oats87 Cool, yes sounds like a good path to me. I saw this a bit late to make changes to the PR before the meeting in 4m, but I'll send an update with that wording after. :+1: :pray:

Reply:
Okay, I sent updated wording (split into second commit). Re-requesting review from everyone.